src/app/commons/page/option/page-item.component.ts

```diff
@@ -0,0 +1,25 @@
+import { ChangeDetectionStrategy, Component, Output, HostListener, Input, EventEmitter, forwardRef } from '@angular/core';
+import { ILuOptionItem, ALuOptionItem } from '@lucca-front/ng';
+import { Page } from '../page.model';
+
+@Component({
+	selector: 'ds-page-item',
+	templateUrl: './page-item.component.html',
+	styleUrls: ['./page-item.component.scss'],
+	changeDetection: ChangeDetectionStrategy.OnPush,
+	providers: [
+		{
+			provide: ALuOptionItem,
+			useExisting: forwardRef(() => DsPageItemComponent),
+			multi: true,
+		},
+	],
+})
+export class DsPageItemComponent<T = Page> extends ALuOptionItem<T> implements ILuOptionItem<T> {
+	@Input() page: T;
+	@Output() onSelect = new EventEmitter<T>();
+	@HostListener('click')
+	onclick() {
+		this.onSelect.emit(this.page);
+	}
```

Comment:
j'ai oublie que pour etendre ALuOptionItem faut aussi implementer `value`

https://github.com/LuccaSA/lucca-front/blob/rc.1.1/packages/ng/libraries/core/src/lib/option/item/option-item.model.ts#L4

rajoute un 
```ts
get value() { return this.page; }
```
et c'est gagne

------------

c'ewst utilise par le option-picker pour trouver l'option-item correspondant a une valeur donnee
 - pour y rajouter la classe `is-selected` pour la valeur courante (c'est pas fait mais ca pourrait etre fait)
 - pour trouver le contenu html a afficher quand t'as pas fournis de _displayer_